modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc

```diff
@@ -0,0 +1,30 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+[id='configuring-managed-secure-boot-in-the-install-config-file_{context}']
+
+= Configuring managed Secure Boot in the `install-config.yaml` file (optional)
+
+To enable managed Secure Boot, add the `bootMode` configuration setting to each node.
+
+[source,yaml]
+.Example
+----
+hosts:
+  - name: openshift-master-0
+    role: master
+    bmc:
+      address: ipmi://<out-of-band-ip>
+      username: <user>
+      password: <password>
+    bootMACAddress: <NIC1-mac-address>
+    hardwareProfile: default
+    bootMode: UEFISecureBoot <1>
+----
+
+<1> The `bootMode` setting is `legacy` by default. Change it to `UEFISecureBoot` to enable managed Secure Boot.
+
+[NOTE]
+====
+See *Node requirements* to ensure the nodes can support managed Secure Boot. If not, you can enable Secure Boot manually, which requires Redfish virtual media.
```

Comment:
I would add a link here to instructions how to set Secure Boot manually

Reply:
They just announced we could put xrefs into the modules. I will try and hopefully they will accept.